services/timeseries.py

```diff
@@ -248,3 +255,15 @@ def delete_repository_data(repository: Repository):
         owner_id=repository.ownerid,
         repo_id=repository.repoid,
     ).delete()
+
+
```

Comment:
Add this new function that is similar to `delete_repository_data` except it only deletes specific `Measurements` (by its name and measurable_id), this will be used when we want to only delete a specific trendline, for example, delete all datapoints of component_id="python3".